libs/mrwebrtc-unityplugin/include/api.h

```diff
@@ -64,6 +69,9 @@ mrsNativeRenderer_GetVideoUpdateMethod() noexcept;
 /// Pipe log entries to Unity's log.
 MR_UNITYPLUGIN__API void MR_UNITYPLUGIN__CALL
 mrsNativeRenderer_SetLoggingFunctions(UnityLogger::LogFunction logDebugFunc,
-                                      UnityLogger::UnityLogger::LogFunction logErrorFunc,
+                                      UnityLogger::LogFunction logErrorFunc,
                                       UnityLogger::LogFunction logWarningFunc);
+
+MR_UNITYPLUGIN__API void MR_UNITYPLUGIN__CALL
+mrsNativeRenderer_SetTextureSizeChangeCallback(TextureSizeChangeCallback callback) noexcept;
```

Comment:
I'd rename to `TextureSizeChanged` (both function and type), I think event names generally use the past tense to indicate the event is raised after the underlying action occurred.